dapr/actor/runtime/actor.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+
+"""
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT License.
+"""
+
+from dapr.actor.actor_interface import ActorInterface
+from dapr.actor.runtime.methodcontext import ActorMethodContext
+from dapr.actor.runtime.context import ActorRuntimeContext
+
+class Actor:
```

Comment:
Use async/await everywhere: https://docs.python.org/3/library/asyncio.html

Reply:
Yeah I switched to async methods. please review the code again.